ReSharper.FSharp/test/data/features/quickFixes/replaceWithInterpolatedString/String 01 - Single specifier.fs.gold

```diff
@@ -0,0 +1,3 @@
+﻿module Module
+
+let s = ${caret}"%d{10}"
```

Comment:
Can we assume it's safe to remove specifiers when an inserted expression is a literal expression?